sandbox/apparmor/apparmor_test.go

```diff
@@ -633,6 +635,27 @@ func (s *apparmorSuite) TestPromptingSupported(c *C) {
 			parserError:    nil,
 			expectedReason: "apparmor parser does not support the prompt qualifier",
 		},
+		{
+			kernelFeatures: []string{"policy:permstable32:allow", "policy:permstable32:deny", "policy:permstable32:prompt", "policy:notify"},
+			kernelError:    nil,
+			parserFeatures: []string{"mqueue", "prompt"},
+			parserError:    nil,
+			expectedReason: "the kernel does not support prompting for file access",
+		},
+		{
```

Comment:
Is an additional test case needed which tests for the presence of the notify/user file exists but it does not contain the `file` attribute? ie. `kernelFeatures: []string{"policy:permstable32:allow", "policy:permstable32:deny", "policy:permstable32:prompt", "policy:notify", "policy:notify:user"}`?

Reply:
Or where it contains some other attribute like `foo`?

Reply:
Great catch, thanks!

Reply:
Technically, `policy:notify:user` will actually ever be included in the list, with the rationale being that `policy:permstable32` is also not included in the list due to the way the permissions are determined, so for historical reasons I didn't change this.

But I can definitely add a case where `policy:notify:user:foo` is in the list but not `policy:notify:user:file`.

Reply:
Thanks.